CODEOWNERS

```diff
@@ -1 +1 @@
-* @GoogleCloudPlatform/blueprint-solutions @imrannayer @belgana
+* @GoogleCloudPlatform/blueprint-solutions @imrannayer
```

Comment:
This is centrally managed will revert. You can change this here https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/d574e98ffa4200091f35ce8200e480792c8e966a/infra/terraform/test-org/org/locals.tf#L499

Reply:
Fixed in cloud foundation toolkit repo